vnext/Microsoft.ReactNative/Fabric/Composition/TextInput/WindowsTextInputComponentView.h

```diff
@@ -131,6 +131,7 @@ struct WindowsTextInputComponentView
   bool m_drawing{false};
   bool m_clearTextOnSubmit{false};
   bool m_multiline{false};
+  bool m_clearTextOnFocus{false};
```

Comment:
You shouldn't need to duplicate the props.  You can already access this using `windowsTextInputProps().clearTextOnFocus`